R/pkg/R/mllib_regression.R

```diff
@@ -84,6 +84,12 @@ setClass("IsotonicRegressionModel", representation(jobj = "jobj"))
 #' # can also read back the saved model and print
 #' savedModel <- read.ml(path)
 #' summary(savedModel)
+#'
+#' # fit tweedie model
+#' require(statmod)
```

Comment:
while it might be more correct, generally people use `library` instead of `require`